src/GeneratedHydrator/CodeGenerator/Visitor/HydratorMethodsVisitor.php

```diff
@@ -44,25 +47,20 @@ class HydratorMethodsVisitor extends NodeVisitorAbstract
     private $reflectedClass;
 
     /**
-     * @var ReflectionProperty[]
+     * @var string[][]
      */
-    private $accessibleProperties;
-
-    /**
-     * @var PropertyAccessor[]
-     */
-    private $propertyWriters = array();
+    private $classPropertyMap = array();
```

Comment:
If a comment is not relevant, simply don't worry about it and defer to merger. I'll probably apply CS changes on merge anyway.